.gitattributes

```diff
@@ -0,0 +1,2 @@
+sdk_generator/outputs/github_parser.json merge=union
```

Comment:
maybe accidentally pushed? I believe we don't need this...

Reply:
yep that was accidental!

Reply:
can you remove this file from PR?

Reply:
@tahigash Sorry forgot to remove, it's gone now.

Reply:
confirmed the update.